webpack.config.js

```diff
@@ -160,7 +160,7 @@ const customizer = {
 
 const WORDPRESS_NAMESPACE = '@wordpress/';
 const BABEL_NAMESPACE = '@babel/';
-const gutenbergPackages = [ '@babel/polyfill', '@wordpress/dom-ready', '@wordpress/i18n', '@wordpress/hooks', '@wordpress/url' ].map(
+const gutenbergPackages = [ '@babel/polyfill', '@wordpress/dom-ready', '@wordpress/i18n', '@wordpress/hooks', '@wordpress/html-entities', '@wordpress/url' ].map(
```

Comment:
@pierlon The `@wordpress/html-entities` is a new dependency that we use which needs to be polyfilled or else an error occurs?

Reply:
It's needed for the onboarding wizard to load in WP 4.9.